system/src/system_network_diagnostics.cpp

```diff
@@ -279,11 +305,13 @@ class NetworkCellularCellGlobalIdentityMobileCountryCodeDiagnosticData
 
     virtual int get(IntType& val)
     {
-        CellularGlobalIdentity cgi{.size = sizeof(CellularGlobalIdentity), .version = CGI_VERSION_LATEST};
-        cellular_global_identity(&cgi, nullptr);
+        cellular_result_t result;
+        CellularGlobalIdentity cgi = {.size = sizeof(CellularGlobalIdentity),
```

Comment:
I'd prefer this was filled in `getCGI()`.

Reply:
done